pages/about.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <title>About Brooke</title>
 </head>
 <body>
-  
+  <center>
+  <section>
+    <h1>About Brooke</h1>
+    <img alt="Black and white photo of Brooke Vlahos" src="https://i.imgur.com/c9g5dyJ.jpg" style="width:400px"/>
```

Comment:
Just note this is a really big photo (much bigger than 400px).  To make this more accessible for those with slower internet connections, you can link to a 400px width photo (it will load faster and look the same).